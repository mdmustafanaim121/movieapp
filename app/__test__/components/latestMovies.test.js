import LatestMovies from '../../components/latestMovies';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Testing Latest Movies', () => {
    it('Should return latest movies snapshot',()=>{
        const result = renderer.create(<LatestMovies />).toJSON()
        expect(result).toMatchSnapshot()
        })
})
