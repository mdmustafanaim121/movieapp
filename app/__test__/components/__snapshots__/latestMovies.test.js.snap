// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Latest Movies Should return latest movies snapshot 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "alignSelf": "stretch",
      "flex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View>
    <RCTScrollView
      data={Array []}
      disableVirtualization={false}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      initialNumToRender={10}
      keyExtractor={[Function]}
      maxToRenderPerBatch={10}
      onContentSizeChange={[Function]}
      onEndReachedThreshold={2}
      onLayout={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      updateCellsBatchingPeriod={50}
      viewabilityConfigCallbackPairs={Array []}
      windowSize={21}
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
